consensus/api/Cargo.toml

```diff
@@ -1,9 +1,10 @@
 [package]
 name = "mobilecoin-api"
-version = "0.1.0"
+version = "0.1.1"
 authors = ["MobileCoin"]
 build = "build.rs"
 edition = "2018"
+links = "mc-consensus-api"
```

Comment:
I'm not familiar with `links` here. How does this fit in to the plan?

https://doc.rust-lang.org/cargo/reference/manifest.html#the-links-field

Reply:
https://doc.rust-lang.org/cargo/reference/build-scripts.html#the-links-manifest-key